### DISCUSSION TITLE
Advantages of VS vs MinGW builds?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4428] [Q&A]
I should probably know this already, but are there any strong reasons, under Windows, to prefer the MinGW build to the Visual Studio build?

I know (or think I know) that slirp support is in the MinGW build, not the VS build, so the MinGW is preferable for anything that involves networking. But are there any other differences worth considering (speed, stability, etc., etc.)?

Apologies if I should have known this already.

**Comment by Torinde** [comment]
The official description, I think is [here](https://github.com/joncampbell123/dosbox-x/blob/master/INSTALL.md#windows-packages-installer-or-portable)

**Comment by emendelson** [comment]
It is. Thank you.
